Source/Upstream: Yes, in v238-stable branch

From 6079d9da0cd351e49e774fe9bf1609caec6b4067 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Mon, 12 Mar 2018 17:45:42 +0900
Subject: [PATCH 37/51] busctl: add missing oom check

(cherry picked from commit cb05d2a5149517ed524f859aa219e27c601d74ad)
---
 src/busctl/busctl.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/busctl/busctl.c b/src/busctl/busctl.c
index f8c43b507..b4ab0d12d 100644
--- a/src/busctl/busctl.c
+++ b/src/busctl/busctl.c
@@ -112,6 +112,9 @@ static int list_bus_names(sd_bus *bus, char **argv) {
         }
 
         merged = new(char*, hashmap_size(names) + 1);
+        if (!merged)
+                return log_oom();
+
         HASHMAP_FOREACH_KEY(v, k, names, iterator)
                 merged[n++] = k;
 
-- 
2.16.3

